commands/FBComponentCommands.py

```diff
@@ -50,13 +51,17 @@ def options(self):
     ]
 
   def args(self):
-    return [ fb.FBCommandArgument(arg='aView', type='UIView*', help='The view to from which the search for components begins.', default='(id)[[UIApplication sharedApplication] keyWindow]') ]
+    return [ fb.FBCommandArgument(arg='aView', type='UIView* or CKComponent*', help='The view or component from which the search for components begins.', default='(id)[[UIApplication sharedApplication] keyWindow]') ]
 
   def run(self, arguments, options):
     upwards = 'YES' if options.upwards else 'NO'
     showViews = 'YES' if options.showViews == 'YES' else 'NO'
 
     view = fb.evaluateInputExpression(arguments[0])
+    if not viewHelpers.isView(view):
+        # assume it's a CKComponent
+        view = fb.evaluateExpression('((CKComponent *)%s).viewContext.view' % view)
```

Comment:
thanks for taking a look and agreed! 
we need to move to that approach when we can write plugins in something more "objective" and something more "C" to simplify dealing with plugins